cmd/delete.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/amp-labs/cli/flags"
+	"github.com/amp-labs/cli/logger"
+	"github.com/amp-labs/cli/request"
+	"github.com/spf13/cobra"
+)
+
+var deleteCmd = &cobra.Command{ //nolint:gochecknoglobals
+	Use:   "integration:delete <integrationId>",
```

Comment:
I'd suggest something like "delete-integration". It would help with readability. Also a few shells these days will parse `something:` as the start of a URI, and will present a link - so using the suggested form will be less likely to be misconstrued.

Reply:
That's a good point, ok let's do delete-integration